source/includes/_prices.md

```diff
@@ -20,22 +22,128 @@ curl "http://api.lettuce.money/prices?to_symbol=ZAR&from_symbols=BTC,USD,MSFT"
   "fromSymbols": [
     {
       "fromSymbol": "BTC",
-      "lastUpdated": "2018-12-31T11:59:01.001Z",
-      "price": 0.000021,
-      "source": "Crypto Compare"
+      "price": 53504.54788657036,
+      "source": "cryptocompare"
+    },
+    {
+      "fromSymbol":"ETH",
+      "price":1841.2815319462345,
+      "source":"cryptocompare"
+    },
+    {
+      "fromSymbol":"XMR",
+      "price":667.5567423230975,
+      "source":"cryptocompare"
+    }
+  ],
+  "errors": []
+}
+```
+
+## Stocks
+
+Price data of a set of stocks symbols (symbols) in a chose currency / symbol (toSymbol), gathered through our price aggregation.
+
+```shell
+curl "http://api.lettuce.money/pricing/stocks?toSymbol=ZAR&symbols=MSFT,TWTR,VOD.JO"
+  -H "Authorization: mypersonalapikey"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "timestamp": "2019-01-01T00:00:01.001Z",
+  "toSymbol": "ZAR",
+  "fromSymbols": [
+    {
+      "fromSymbol":"MSFT",
+      "price":1532.7689388410452,
+      "source":"alphavantage"
+    },
+    {
+      "fromSymbol":"TWTR",
+      "price":481.9734345351044,
+      "source":"alphavantage"
+    },
+    {
+      "fromSymbol":"VOD.JO",
+      "price":13399.0000,
+      "source":"alphavantage"
+    }
+  ],
+  "errors": []
+}
+```
+
+## Forex
+
+Price data of a set of forex symbols (symbols) in a chose currency / symbol (toSymbol), gathered through our price aggregation.
+
+```shell
+curl "http://api.lettuce.money/pricing/forex?toSymbol=ZAR&symbols=EUR,GBP,USD"
+  -H "Authorization: mypersonalapikey"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "timestamp": "2019-01-01T00:00:01.001Z",
+  "toSymbol": "ZAR",
+  "fromSymbols": [
+    {
+      "fromSymbol":"EUR",
+      "price":16.730801405387318,
+      "source":"cryptocompare"
+    },
+    {
+      "fromSymbol":"GBP",
+      "price":18.85369532428356,
+      "source":"cryptocompare"
+    },
+    {
+      "fromSymbol":"USD",
+      "price":14.63057790782736,
+      "source":"cryptocompare"
+    }
+  ],
+  "errors": []
+}
+```
+
+## Forex
+
```

Comment:
I don't understand what you mean by "portfolio" in this context. 
The heading still reads "Forex".
Is this the call that allows any type of symbol? If yes, then why not just always call this endpoint? Why have specific /forex /cryptocurrencies points at all?